src/compile/data/assemble.ts

```diff
@@ -200,6 +200,12 @@ function makeWalkTree(data: VgData[]) {
       dataSource.transform = dataSource.transform.concat(node.assemble());
     }
 
+    if (node instanceof AggregateNode) {
```

Comment:
Yes I can. We need it because if we do not then the vega spec has a null name for one of the data objects but the vega-schema.json only allows for string names.